source/extensions/omni.isaac.lab/omni/isaac/lab/envs/mdp/actions/task_space_actions.py

```diff
@@ -115,6 +115,19 @@ def raw_actions(self) -> torch.Tensor:
     def processed_actions(self) -> torch.Tensor:
         return self._processed_actions
 
+    @property
+    def jacobian_w(self) -> torch.Tensor:
+        return self._asset.root_physx_view.get_jacobians()[:, self._jacobi_body_idx, :, self._joint_ids]
```

Comment:
This currently undoes the fix of https://github.com/isaac-sim/IsaacLab/pull/1033.
```suggestion
        return self._asset.root_physx_view.get_jacobians()[:, self._jacobi_body_idx, :, self._jacobi_joint_ids]
```